src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -27,27 +27,7 @@
 import org.bukkit.World;
 import org.bukkit.World.Environment;
 import org.bukkit.block.Block;
-import org.bukkit.entity.Animals;
-import org.bukkit.entity.Creature;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.EntityType;
-import org.bukkit.entity.Explosive;
-import org.bukkit.entity.FallingBlock;
-import org.bukkit.entity.Horse;
-import org.bukkit.entity.Item;
-import org.bukkit.entity.LightningStrike;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.entity.Monster;
-import org.bukkit.entity.Player;
-import org.bukkit.entity.Projectile;
-import org.bukkit.entity.Rabbit;
-import org.bukkit.entity.Tameable;
-import org.bukkit.entity.ThrownPotion;
-import org.bukkit.entity.Vehicle;
-import org.bukkit.entity.WaterMob;
-import org.bukkit.entity.Llama;
-import org.bukkit.entity.Donkey;
-import org.bukkit.entity.Mule;
+import org.bukkit.entity.*;
```

Comment:
Please do not use wildcard imports.

Reply:
Sorry, IntelliJ does that all by itself, I didn't touch those

Reply:
You can configure it not to do that, but yea the defaults do that unfortunately.

Reply:
Duly noted, thanks for letting me know. 
I fixed it so all is good again :)